core.js

```diff
@@ -3409,11 +3409,11 @@ const tickAnimationFrame = () => {
   }
 };
 
-const _makeRequestAnimationFrame = window => (fn, priority = 0) => {
+const _makeRequestAnimationFrame = (window, prio = 0) => (fn, priority = prio) => {
```

Comment:
This thing can take a device string and decorate it on the `fn` via `Symbol`.

Reply:
In addition to the prio? That might work.